tools/kompos/index.html

```diff
@@ -299,8 +311,8 @@
 
 <div id="code-views" class="card-columns"></div>
 
-<div class="card system-status" data-spy="affix">
-<div id="debugger" class="card-body">
+<div class="card system-status">
```

Comment:
I removed it because it does not scroll the elements inside the div. I'm not sure why does not work, I read this property should lock the div in a specific area and enable scrolling. The solution I found was to add a css style to the div (.div-scroll), specifying a height and enabling scrolling vertically. I think this is needed because if we do not specify a fixed size for the debugger div, it can overlap the activity that is below it. As consequence, the user will not be able to press the buttons of the activity.